packages/web/src/components/Settings.tsx

```diff
@@ -10,9 +25,14 @@ export default () => {
             ...settings,
         },
     });
-
+    const [modelOptions, setModelOptions] = React.useState([
+        { label: "GPT 3.5", value: "gpt-3.5-turbo" },
+        { label: "GPT 4", value: "gpt-4" },
+        { label: "GPT 4 (32k)", value: "gpt-4-32k" },
+    ]);
     const handleSubmit = form.onSubmit((values) => {
         setSettings(values);
+        closeAllModals();
```

Comment:
There's probably a better way to close this modal on save

Reply:
There's a way to specify a modal by ID (which there isn't for this one), but this is fine. I rarely have more than one modal opened at a time, so I doubt this will cause an issue in the future.